pkg/cmd/experimental/syncgroups/interfaces/interfaces.go

```diff
@@ -35,3 +35,8 @@ type LDAPGroupListerNameMapper interface {
 	LDAPGroupLister
 	LDAPGroupNameMapper
 }
+
+// LDAPGroupLocator determines if a group identified by an LDAP group UID exists on the LDAP server
+type LDAPGroupLocator interface {
```

Comment:
`LDAPGroupExistenceChecker` or something.  a `Locator` would tell me where somethign is.